### ISSUE TITLE
Launching Edge with --user-data-dir= results in Blocked Unsafe Flag

### ISSUE THREAD

**Issue by ohouigheighi** [rcmaehl/MSEdgeRedirect#463]
### Preflight Checklist

- [ ] Microsoft Edge is selected in any "How do you want to open this?" box
- [ ] I've ran the [Cleanup tool](https://raw.githubusercontent.com/rcmaehl/MSEdgeRedirect/main/Assets/Cleanup%20Tool.ps1) and tried another install

### Install Type

New Install

### Install Mode

Active Mode

### ℹ Behavior

Edge will not launch when using --user-data-dir=
For example, my Edge shortcut's Target is "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe" --user-data-dir="C:\Edge-data\test"
When I attempt to launch Edge with that shortcut, Edge does not launch - nothing happens.
MSEdgeRedirect's AppSecurity.log logs "Blocked Unsafe Flag: --user-data-dir=C:\Edge-data\test"

Is there a way to allow the --user-data-dir flag?
Thanks

### Microsoft Windows version

Version 23H2 (OS Build 22631.4169)

### Other Software

MSEdgeRedirect 0.7.5.3 Update
msedge.exe Version 129.0.2792.52 (Official build) (64-bit)

**Comment by rcmaehl** [comment]
If you create a registry entry in `HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Robert Maehl Software\MSEdgeRedirect` called `RunUnsafe` as a DWORD, with a value of 1 you can bypass the check but this allows all flags.

**Comment by ohouigheighi** [comment]
Under `HKEY_LOCAL_MACHINE\SOFTWARE\Policies` I do not have a `Robert Maehl Software` Key. I do have a `HKEY_LOCAL_MACHINE\SOFTWARE\Robert Maehl Software\MSEdgeRedirect` Key, so I tried creating the DWORD with value 1 called `RunUnsafe` under that existing path, (`HKEY_LOCAL_MACHINE\SOFTWARE\Robert Maehl Software\MSEdgeRedirect`), but that didn't work.

Then I tried creating the Key and DWORD in the location you specified, `HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Robert Maehl Software\MSEdgeRedirect` but that did not work either.

The same messages are logged in AppSecurity.log. Appreciate the help.
